packages/fxa-settings/src/pages/InlineRecoverySetup/index.tsx

```diff
@@ -0,0 +1,233 @@
+  const localizedDownloadButtonTitle = ftlMsgResolver.getMsg(
+    'inline-recovery-download-button',
+    'Download'
+  );
+  const localizedPrintButtonTitle = ftlMsgResolver.getMsg(
+    'inline-recovery-print-button',
+    'Print'
+  );
+  const localizedGraphicAriaLabel = ftlMsgResolver.getMsg(
+    'inline-recovery-setup-image-description',
+    'Document that contains hidden text.'
+  );
+
+  /* TODO: - Add in copy/download/print/continue/cancel actions for all buttons
+   *       - Add in metrics for all events
+   *       - Add tests for all metrics
+   */
+  const printCode = () => {};
+  const copyCode = () => {};
+  const downloadCode = () => {};
+  const cancelSetup = () => {};
+  const continueSetup = () => {};
+
+  return (
+    <>
+      {showConfirmation ? (
+        <>
+          <div className="mb-4">
+            {serviceName ? (
+              <FtlMsg id="inline-recovery-confirmation-header">
+                <h1 className="card-header">
+                  Confirm backup authentication code{' '}
+                  <span className="card-subheader">{`to continue to ${serviceName}`}</span>
+                </h1>
+              </FtlMsg>
+            ) : (
+              <FtlMsg id="inline-recovery-confirmation-header-default">
+                <h1 className="card-header">
+                  Confirm backup authentication code{' '}
+                  <span className="card-subheader">
+                    to continue to account settings
+                  </span>
+                </h1>
+              </FtlMsg>
+            )}
+          </div>
+          <section>
+            <form noValidate>
+              <div>
+                <RecoveryCodesGraphic
+                  role="img"
+                  aria-label={localizedGraphicAriaLabel}
+                  className="mx-auto"
+                />
+                <FtlMsg id="inline-recovery-confirmation-description">
+                  <p className="text-sm mb-6">
+                    To ensure that you will be able to regain access to your
+                    account in the event of a lost device, please enter one of
+                    your saved backup authentication codes.
+                  </p>
+                </FtlMsg>
+                <InputText
+                  label={localizedInputTextLabel}
+                  className="tooltip-below recovery-code text-start my-4"
+                  anchorStart={true}
+                  required
+                  autoFocus
+                  autoComplete="off"
+                />
+                <FtlMsg id="inline-recovery-confirm-button">
+                  <button type="submit" className="cta-primary cta-xl w-full">
+                    Confirm
+                  </button>
+                </FtlMsg>
+                <div className="flex justify-between mt-4">
+                  <FtlMsg id="inline-recovery-back-link">
+                    <button className="link-blue text-sm">Back</button>
```

Comment:
We may want these buttons to have `type="button"`, the default is `submit` and I think it's best practice to always specify the `type`.